app/controllers/courses_controller.rb

```diff
@@ -22,8 +22,9 @@ def update
     if @course.update(course_params)
       flash[:success] = 'Your changes have been saved'
       redirect_to(
-        provider_course_path(
+        provider_recruitment_cycle_course_path(
           @course.provider_code,
+          @course.recruitment_cycle,
```

Comment:
I know this is a pain, but considering at some point we're possibly going to actually want the `recruitment_cycle` object in mcfe, should we be using `@course.recruitment_cycle_year` here?